presto-main/src/test/java/io/prestosql/sql/planner/assertions/BasePushdownPlanTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.sql.planner.assertions;
+
+import io.prestosql.Session;
+import io.prestosql.metadata.QualifiedObjectName;
+import io.prestosql.metadata.TableHandle;
+import io.prestosql.spi.connector.ColumnHandle;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Extension of {@link BasePlanTest} that provides connector metadata methods
+ */
+public abstract class BasePushdownPlanTest
```

Comment:
This seems that it does not belong to this commit. Maybe you can extract this to separate commit or to the commit that is introducing the usage of it.

Reply:
added as a separate commit.